.changeset/afraid-hats-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/fields-document': patch
+---
+
+refine the panel render styles
```

Comment:
Btw, the document field is currently not being published so you don't need to add changesets for it.

Reply:
Gotcha, cheers